edid-parse.c

```diff
@@ -26,42 +26,33 @@
 #include <stdlib.h>
 #include <string.h>
 
-#define TRUE 1
-#define FALSE 0
-
-static int decode_header(const uchar *edid)
+static int is_edid_header(const uchar *edid)
 {
 	if (memcmp(edid, "\x00\xff\xff\xff\xff\xff\xff\x00", 8) == 0)
-		return TRUE;
-	return FALSE;
+		return 1;
+	return 0;
 }
 
 static void decode_lf_string(const uchar *s, int n_chars, char *result)
 {
 	int i;
-	for (i = 0; i < n_chars; ++i)
-	{
-		if (s[i] == 0x0a)
-		{
+
+	for (i = 0; i < n_chars; ++i) {
+		if (s[i] == 0x0a) {
 			*result++ = '\0';
 			break;
-		}
-		else if (s[i] == 0x00)
-		{
+		} else if (s[i] == 0x00) {
 			/* Convert embedded 0's to spaces */
 			*result++ = ' ';
-		}
-		else
-		{
+		} else {
 			*result++ = s[i];
 		}
 	}
 }
 
-static void decode_display_descriptor(const uchar *desc, MonitorInfo *info)
+static void decode_display_descriptor(const uchar *desc, MonitorInfo * info)
```

Comment:
Here's one of the caveats with GNU indent.  When it doesn't know of a composed type it inserts a gratuitous extra space here.  It's soooooo annoying and the only two ways of getting around it, that I know of is:

1. Add the type to the indent command line, `indent -T MonitorInfo ...`, I've added a few to my own wrapper script, which you may have noticed.
2. Do it like the kernel does, which is what I'm beginning to lean more and more to myself, i.e. don't use `typedef` but instead use `struct MonitorInfo` everywhere, because *that* GNU indent understands and does right by ... *sigh*